.docker/test-OpenData.ini

```diff
@@ -29,6 +29,7 @@ use = egg:ckan
 full_stack = true
 cache_dir = /tmp/%(ckan.site_id)s/
 beaker.session.key = ckan
+beaker.session.type = file
```

Comment:
was this defaulting to file before?

Reply:
I think so. In higher environments it's configured to use Redis.